### DISCUSSION TITLE
Elsa2 dashboard not loading any activities

### DISCUSSION THREAD

**Discussion by SachinMehndiratta** [elsa-workflows/elsa-core#949] [Q&A]
Hello,

Elsa2 dashboard is not loading any activities on dashboard, I updated my solution from Elsa 1 Latest stable and made changes to activity definition as required.

Activities are not appearing on new dashboard, any clues?

**Comment by sfmskywalker** [comment]
It’s hard to say without seeing the project. Are you hosting the dashboard and server on the same project? Or as two separate projects? I’m working on QuickStart documentation in the elsa-website repo that might be of help?

**Reply by SachinMehndiratta** [reply]
Have a class library project where activities are defined, this library is referenced to web api which is acting as dashboard project. So a monolith arrangement it is.

I believe documentation will help for sure, nevertheless a hello world activity on dashboard as a sample would serve better.

**Reply by SachinMehndiratta** [reply]
@sfmskywalker - I got this working by creating a monolith project from scratch, kind of replicating the sample project. Earlier the  Elsa.Activities... were indirect references in my Server coming from Host project. Also, found the reference to Elsa was missing in Server application.

There wasn't any error earlier, only the activities popup on dashboard was coming blank. Adding the Elsa, Esla.Activities. packages to Server project resolves this.

**Reply by sfmskywalker** [reply]
Nice. Meanwhile, I've been working on a couple of quick start guides that may help as well (although it looks like you have figured it out already, which is awesome): https://elsa-workflows.github.io/elsa-core/docs/next/quickstarts/quickstarts-aspnetcore-server-dashboard-and-api-endpoints
